docs/setup.rst

```diff
@@ -0,0 +1,56 @@
+=====
+Setup
+=====
+
+Setting up scripts for OMERO.web has to be done by an administrator of your \
```

Comment:
```suggestion
Setting up server-side scripts for OMERO has to be done by an administrator of your \
```

Reply:
the scripts are not bound to OMERO.web. They will also be available from OMERO.insight.